docs/prometheus-exporter.md

```diff
@@ -26,3 +26,69 @@ The following options are available for configuring the pushing of metrics:
+
+Note: This metric is updated during "housekeeping", meaning the update frequency depends on the node configuration `--housekeeping-interval` (`CONCORDIUM_NODE_CONNECTION_HOUSEKEEPING_INTERVAL`) which is one every 30 seconds by default.
+
+### `network_sent_bytes`
+
+Total number of bytes sent over the network. Only network message sent to connected peers are accounted.
+
+Note: This metric is updated during "housekeeping", meaning the update frequency depends on the node configuration `--housekeeping-interval` (`CONCORDIUM_NODE_CONNECTION_HOUSEKEEPING_INTERVAL`) which is one every 30 seconds by default.
+
+### `network_packets_received_total`
+
+Total number of network packets received from peers. This is accounted before the any form of deduplication.
+
+### `network_packets_sent_total`
+
+Total number of network packets sent to peers.
+
+### `network_connected_peers`
+
+Current number of connected peers. This is incremented when a peer completes a handshake and decremented again when the connection is dropped.
+
+###  `network_connections_received_total`
+
+Total number of connections received. Incremented everytime someone tries to establish a new connection, meaning even the failed connections are accounted, such as when the address is banned, duplicate connection or the node is at its limit on number of connections.
+
+### `network_inbound_high_priority_message_drops_total`
+
+Total inbound high priority messages dropped due to the queue for high priority messages being full.
+See `network_inbound_high_priority_message_queue_size` for the current size of the queue.
+
+### `network_inbound_low_priority_message_drops_total`
+
+Total inbound low priority messages dropped due to the queue for low priority messages being full.
+See `network_inbound_low_priority_message_queue_size` for the current size of the queue.
+
+### `network_inbound_high_priority_messages_total`
+
+Total inbound high priority messages received. This is incremented when a consensus message is enqueue in the high priority queue. Note: Does not include the dropped messages, see `network_inbound_high_priority_message_drops_total`.
+
+### `network_inbound_low_priority_messages_total`
+
+Total inbound low priority messages received. This is incremented when a consensus message is enqueue in the low priority queue. Note: Does not include the dropped messages, see `network_inbound_low_priority_message_drops_total`.
+
+### `network_inbound_high_priority_message_queue_size`
+
+Current number of consensus messages in the inbound high priority queue. Start dropping messages when larger than 16 * 1024.
+
+### `network_inbound_low_priority_message_queue_size`
+
+Current number of consensus messages in the inbound low priority queue. Start dropping messages when larger than 32 * 1024.
+
+### `network_outbound_high_priority_message_queue_size`
+
+Current number of consensus messages in the outbound high priority queue. Start dropping messages when larger than 8 * 1024.
```

Comment:
Also this should generally be around 0. Some spikes are fine, but if this is large then there is an indication that there are too few resources.

Reply:
I have added the same comment for all 4 of the metrics related to message queue size.